common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -6,7 +6,9 @@
 %>
 <form class="choicegroup capa_inputtype" id="inputtype_${id}">
     <fieldset ${describedby}>
-      <legend id="${id}-legend" class="response-fieldset-legend field-group-hd">${response_data['label']}</legend>
+      % if response_data['label']:
```

Comment:
Why do we need this check ? we need legend anyway i.e. with or without value. is't it?